docs/content/docs/ai-mocks/openai.md

```
@@ -17,12 +17,12 @@ Currently, it supports:
 
 ## Quick Start
 
-Add Dependency Include the library in your test dependencies (Maven or Gradle).
+Include the library in your test dependencies (Maven or Gradle).
 
 {{< tabs "dependencies" >}}
 {{< tab "Gradle" >}}
 ```kotlin
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)